apps/server/src/modules/h5p-library-management/service/h5p-library-management.service.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-console */
```

Comment:
please remove this eslint exception.

Reply:
Solved: 45ef16a8b948ff373855405b9858c2c79a5dd717